requirements/static.txt

```diff
@@ -0,0 +1,28 @@
+# SHA1:5a0b1bb22ae805d8aebba0f3bf05ab91aceae0d8
+#
+# This file is autogenerated by pip-compile-multi
+# To update, run:
+#
+#    pip-compile-multi
+#
+cfgv==3.3.1
+    # via pre-commit
+distlib==0.3.6
+    # via virtualenv
+filelock==3.9.0
+    # via virtualenv
+identify==2.5.18
+    # via pre-commit
+nodeenv==1.7.0
+    # via pre-commit
+platformdirs==3.1.0
+    # via virtualenv
+pre-commit==3.1.1
+    # via -r static.in
+pyyaml==6.0
+    # via pre-commit
+virtualenv==20.20.0
+    # via pre-commit
+
+# The following packages are considered to be unsafe in a requirements file:
+# setuptools
```

Comment:
We also have a `wheels.in` file for when we build the python package, if you want to add that now that you're are it.